tests/test_handle_files.py

```diff
@@ -0,0 +1,44 @@
+import unittest
+from os.path import abspath, dirname, join
+
+from kodi_addon_checker.common import relative_path
+from kodi_addon_checker.handle_files import find_file
+from kodi_addon_checker.handle_files import create_file_index
+from kodi_addon_checker.handle_files import find_file_recursive as FFR
+
+HERE = abspath(dirname(__file__))
+
+
+class TestFindFile(unittest.TestCase):
+
+    def setUp(self):
+        self.path = "/tmp"
```

Comment:
`self.path = tempfile.mkdtemp()`, directory has to be deleted explicitly at the end of the test